test/integration/user_signup_test.rb

```diff
@@ -103,22 +95,20 @@ class CreateAUserTest < ActionDispatch::IntegrationTest
     fill_in "signup[password]", with: "password"
     fill_in "signup[password_confirmation]", with: "password"
     fill_in "signup[email]", with: "email@example.com"
+    check "signup[tenant]"
     click_link_or_button "Create Account"
+    fill_in "tenant_signup[location]", with: "location"
+    fill_in "tenant_signup[organization]", with: "organization"
+    click_link_or_button "Create Organization"
 
     assert_equal root_path, current_path
-    assert page.has_content?("Thank you for creating an account.")
+    assert page.has_content?("Organization Created.")
```

Comment:
Maybe assert that a more descriptive message is displayed? Maybe "Your organization, #{org}  has been successfully created. You will receive an email at #{user.email} when your org has been approved to request loans"